scoap3/modules/compliance/compliance.py

```diff
@@ -93,23 +93,27 @@ def __find_regexp_in_pdf(extra_data, patterns, forbidden_patterns=None, accept_e
 def _files(record, extra_data):
     """Check if it has the necessary files: .xml, .pdf, .pdfa """
 
-    file_types = [file['filetype'] for file in record.get('_files', ())]
+    journal = get_first_journal(record)
+    required_files = current_app.config.get('COMPLIANCE_JOURNAL_FILES', {}).get(journal)
 
-    check_accepted = True
-    details = ''
+    if not required_files:
+        return True, ('No required files defined!', ), None
+
+    available_files = {file['filetype'] for file in record.get('_files', ())}
```

Comment:
`file['filetype']` can this fail?

Reply:
Shouldn't, but I'll replace it with `get` to make it more secure.